### PR TITLE
add the currency management

### PR DESCRIPTION
**Pull request by sona-podholova**
# TL;DR

Ability to manage currencies

# Proof
image

# Merge request checklist

Please check if your merge request fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been reviewed and added/updated if needed (for bug fixes/features)
- [x] Build (`yarn build`) was run locally and any changes were pushed for both API and clients
- [x] API (`yarn api:dev`) runs locally and any fixes were made for failures
- [x] Admin Client (`cd clients/admin && yarn start`) runs locally and any fixes were made for failures

# Merge request type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no APIoduce  changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Before this merge request
Currencies were missing

# How you fixed what was wrong
It is possible to add and edit currencies

### CODE REVIEW

File: clients/admin/src/components/currencies/CurrencyList.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    import DataGrid from '../../molecules/datagrid/Datagrid.svelte';
+    import type { RowAction } from 'src/molecules/datagrid/types';
+    import FirstColumn from './FirstColumn.svelte';
+    import type { CurrencyListPartsFragment } from '../../generated/graphql';
+
+    export let currencies: CurrencyListPartsFragment[] | undefined;
+
+    const columns = [
+        {
+            name: 'Bla',
```
Comment (davidpodhola) [inline clients/admin/src/components/currencies/CurrencyList.svelte:11]: This should be fixed, I know we have it everywhere, more a reminder for me :)

**Review by davidpodhola — APPROVED** [review]
